packages/issue-utils/src/issue-comments.ts

```diff
@@ -0,0 +1,104 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// eslint-disable-next-line node/no-extraneous-import
+import {Octokit} from '@octokit/rest';
+
+/**
+ * It creates a comment string used for `addOrUpdateissuecomment`.
+ */
+export const getCommentMark = (installationId: number): string => {
+  return `<!-- probot comment [${installationId}]-->`;
```

Comment:
I wonder if it's worth including a timestamp or updating a counter for how many times it's commented, to make sure it's run recently.